### PR TITLE
Fix Windows Compiler Warnings

### PR DESCRIPTION
**Pull request by stsantilena**
These two functions were added for the missing initial directory bug (#385). This fix currently only applies to Linux. As a result, when compiling on Windows, the compiler throws a warning about the functions not being used.

### CODE REVIEW

**Review by dkhokhlov — APPROVED** [review]
lgtm

File: common/fs/src/cache/dir_path.rs [hunk 1]
```diff
@@ -89,6 +89,7 @@ impl From<DirPathBuf> for PathBuf {
 ///
 /// When a valid directory is found, it adds the missing piece of the directory
 /// path to `postfix` and returns the `DirPathBuf` result.
+#[cfg(unix)]
```
Comment (dkhokhlov) [inline common/fs/src/cache/dir_path.rs:92]: @stsantilena 
i missed it - we want this feature (non-existing directories) tested on windows too.
possible?

**Review by dkhokhlov — CHANGES_REQUESTED** [review]

File: common/fs/src/cache/dir_path.rs (hunk 1 above)
Comment (stsantilena) [reply common/fs/src/cache/dir_path.rs:92]: Not at this time. The fix for missing directory only applies to Linux at the moment. There is a ticket to extend to Windows.

**Review by dkhokhlov — APPROVED** [review]
lgtm

**Pull request merged**